README.md

```diff
@@ -1,24 +1,154 @@
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+The Recipe App was built with:
 
-* Database initialization
+    - Ruby On Rails
+    - PostgreSQL
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+</details>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+</details>
 
-* ...
+<details>
+<summary>Database</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add posts**
+- **Add comments**
+- **Display details**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To run this project, you'll need to install:
+
+    - Ruby 2.7.0 or later
+    - Rails 6.0 or later
+    - PostgreSQL
+
+### Installation
+
+Install this project with:
+
+```
+    Clone this repository to your desired folder:
+
+`git@github.com:jonathanmunamire/Recipe-App.git`
+
+cd recipe-app
+
+    Install the required gems:
+
+bundle install
+
+    Set up the database:
+
+rails db:create
+
+rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+bundle exec rspec
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  Not yet deployed
+```
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jonathan Munamire**
+
+- GitHub: [@jonathanmunamire](https://github.com/jonathanmunamire)
+- Twitter: [@amanimunamire](https://twitter.com/amanimunamire)
+- LinkedIn: [Jonathan Munamire](https://www.linkedin.com/in/jonathanmunamire/)
+
+👤 **Abdullateef Bello**
+
+- GitHub: [@Ibnballo1](https://github.com/Ibnballo1/)
+- Twitter: [@webprotekh](https://twitter.com/webprotekh)
+- LinkedIn: [Abdullateef Bello](https://www.linkedin.com/in/abdullateef-bello-1b8006228/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add a meal planner to help users plan their weekly meals.
+- Allow users to add pictures to their recipes.
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ajonathanmunamire/Recipe_App.git/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...give it a ⭐️!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [@microverse](https://www.microverse.org/)
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/jonathanmunamire/Recipe_App/blob/e8aa46d19e1f0e413df26a25a6eaa1b0f1e28068/LICENSE) licensed.
+
```

Comment:
- [ ] The link to the MIT license is broken. It is better to have a link that goes to the MIT license. Kindly fix this issue